### PR TITLE
Add tabs section

### LINKED ISSUE #17: Make the "Visión, misión, nuestra identidad" component.

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR adds the "tabs" section, a section that allows the Prismic users to create selectable tabs and customize them.
This PR closes #17.

Desktop:

tabs

Mobile:
I kept the long borders when selecting the items to divide each item in a more significant way.
responsive-tabs

### CODE REVIEW

File: components/sections/Tabs/Tabs.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import "../../../styles/fonts.scss";
+
+.tabs {
+  min-height: 300px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  font-family: $poppins-semibold;
+}
+
+.tabsName {
```
Comment (hernaneg350) [inline components/sections/Tabs/Tabs.scss:10]: Use underscores. They will be converted to camel case automagically.

File: components/sections/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              {tab.name}
+            </div>
+          )}
+        </div>
+        <div className={styles.tabsContent}>
+          {tabs_list.map((tab, index) =>
+            <div
+              className={styles.tabTitle}
+              style={{
+                color: tabs_content_color ?? "#ffffff",
```
Comment (hernaneg350) [inline components/sections/Tabs/Tabs.tsx:66]: Should we use a class "active" for this?

Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.tsx:66]: I've decided to leave the tabs content, name and background color customizable so they can change it at will. The customization prevents us from having predefined classes :(

File: components/sections/Tabs/Tabs.scss (hunk 1 above)
Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.scss:10]: I wouldn't have a way of writing classes names differently from the way we access them. We could decide if we want everything with underscore or with camel case.

Comment (hernaneg350) [reply components/sections/Tabs/Tabs.scss:10]: Sorry, I meant dashes. CSS convention is to use dashes. Javascript convention is to use camel case. I prefer to keep each language convention, and the framework offers a tool for it, which means that conversion from dashes to camel case is a convention imposed by the framework as well.

File: components/sections/Tabs/Tabs.tsx (hunk 2 above)
Comment (hernaneg350) [reply components/sections/Tabs/Tabs.tsx:66]: I don't understand how putting this in a class prevents us to change it at will. The reason is to keep a single source to look styles in.

Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.tsx:66]: I've made the section totally customizable in a color level. The prismic user can select the different colors at will.
Are you saying that we should put the default into classes?

File: components/sections/Tabs/Tabs.scss (hunk 1 above)
Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.scss:10]: CSS doesn't have any conventions on how to write class names. In the FD UI we used camel case for every class.
It is true that CSS doesn't have camel case per se, as it's case insensitive, but camel casing is only used by us to read it properly.

File: components/sections/Tabs/Tabs.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  padding-left: 15vw;
+  padding-right: 15vw;
+  @media (max-width: 500px) {
+    padding-left: 10vw;
+    padding-right: 10vw;
+  }
+}
+
+.selectedTabName {
+  border-bottom-style: solid;
```
Comment (meelrossi) [inline components/sections/Tabs/Tabs.scss:45]: you can replace all this with`border-bottom: 3px solid #df8832`

File: components/sections/Tabs/Tabs.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  font-size: 20px;
+  cursor: pointer;
+  margin-right: 30px;
+  color: #81a88f;
+  margin-left: 30px;
+  padding-bottom: 5px;
+}
+
+.tabsContent {
+  padding-left: 15vw;
```
Comment (meelrossi) [inline components/sections/Tabs/Tabs.scss:36]: `padding: 0 15vw` 

i know im being a pain in the ass its just a recommendation to have simpler css but you can leave it like this if you want

File: components/sections/Tabs/Tabs.scss (hunk 3 above)
Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.scss:45]: You're right, I kept adding styles and I didn't realize that I had the whole set.

File: components/sections/Tabs/Tabs.scss (hunk 4 above)
Comment (LautaroPetaccio) [reply components/sections/Tabs/Tabs.scss]: That's a perfectly valid comment, thanks!

File: components/sections/Tabs/Tabs.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  flex-direction: row;
+  padding-bottom: 30px;
+  justify-content: center;
+  align-content: center;
+  @media (max-width: 500px) {
+    flex-direction: column;
+  }
+}
+
+.tabTitle {
```
Comment (hernaneg350) [inline components/sections/Tabs/Tabs.scss:22]: camel

**Pull request merged**